examples/out_lib/out_lib.c

```diff
@@ -59,32 +96,35 @@ int main()
     in_ffd = flb_input(ctx, "lib", NULL);
     flb_input_set(ctx, in_ffd, "tag", "test", NULL);
 
-    /* Register my callback function */
+    /* Register output callback functions */
 
     /* JSON format */
-    callback.cb = my_stdout_json;
-    callback.data = NULL;
+    callback_json.cb = my_stdout_json;
+    callback_json.data = NULL;
 
-    out_ffd = flb_output(ctx, "lib", &callback);
+    out_ffd = flb_output(ctx, "lib", &callback_json);
     flb_output_set(ctx, out_ffd, "match", "test", "format", "json", NULL);
 
     /* Msgpack format */
-    /*
-    out_ffd = flb_output(ctx, "lib", my_stdout_msgpack);
+    callback_msgpack.cb = my_stdout_msgpack;
+    callback_msgpack.data = NULL;
+    out_ffd = flb_output(ctx, "lib", &callback_msgpack);
```

Comment:
This line overwrites `out_ffd`.
It may be better to save as `out_ffd_msgpack` or something.

Reply:
Done.